shell/common/BUILD.gn

```diff
@@ -141,6 +142,18 @@ source_set("common") {
   public_configs = [ "//flutter:config" ]
 }
 
+source_set("surface_frame") {
```

Comment:
Why is this in its own target? The only place this is being included (flow) also includes common.

Reply:
To workaround a circular dependency. `:common` depends on `//flutter/flow`, but `//flutter/flow` depends on `:surface_frame`.